### ISSUE TITLE
Object Oriented CSS (less or sass)

### ISSUE THREAD

**Issue by hvorragend** [zikula/core#354]
I thought that we can make Zikula more attractive if we support Object Oriented CSS (http://lesscss.org or http://sass-lang.com/).

This kind of dynamic and programmable stylesheets seems currently to gain a large community.

It would also be an advantage for us because we could then (again) configure in the colors of a theme in the theme administration backend.

We can pass these color values then to the stylesheet and are thus very flexible. Of which would also benefit the global core stylesheet and it will be a little more colorful.

_Example palette:_

````
[palette1]
bgcolor = #F9F9F9
color1 = #739FC4
color2 = #A8C4DB
color3 = #D4E2ED
color4 = #116DAA
color5 = #0059AA
color6 = #0D5492
color7 = #004078
color8 = #004078
sepcolor = #000000
link = #4C5EA8
vlink = #4C5EA8
hover = #006699
````

**Comment by phaidon** [comment]
I reopened this issue because less has much more options, then in this topic discused. Here an overview of functions we dont use yet:

1. Less gives the possibility to change bootstrap font-awesome and other css by variables.
2. Less gives the possibility to create a combined css (core.css, font-awesome and bootstrap for example)
3. Less gives the possibility to disable components in 3-party components (e.g. glyphicons in bootstrap)

**Comment by craigh** [comment]
ah, but you did not re-open it :wink:

**Comment by craigh** [comment]
This was implemented in Core-1.4.0
